install.sh

```diff
@@ -7,7 +7,13 @@ nvm_has() {
 }
 
 nvm_install_dir() {
-  command printf %s "${NVM_DIR:-"$HOME/.nvm"}"
+  if [ ! -z "$NVM_DIR" ]; then
+    printf %s "${NVM_DIR}"
+  elif [ ! -z "$XDG_CONFIG_HOME" ]; then
+    printf %s "${XDG_CONFIG_HOME/nvm/}"
+  else
+    printf %s "${HOME/.config/nvm/}"
```

Comment:
inside the home dir, it *must* be `.nvm` directly inside `$HOME` - there's no `.config` directory convention i'm interested in or aware of.